### PR TITLE
Logstash simple installing

### PR DESCRIPTION
**Pull request by Karina5005**
#1

### CODE REVIEW

**Comment by Karina5005** [comment]
Pull created by EnergyNick at 2022-11-11T13:26:54Z

File: tasks/configuration.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (a-alex-l) [inline tasks/configuration.yml:1]: It's better to move this task to handlers. Isn't it?

File: defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (a-alex-l) [inline defaults/main.yml:1]: Where does certificate configuration present on this pr?

File: tasks/configuration.yml (hunk 1 above)
Comment (a-alex-l) [inline tasks/configuration.yml:1]: I can't see any files above in this pr

File: defaults/main.yml (hunk 2 above)
Comment (a-alex-l) [inline defaults/main.yml:1]: Where is a beats configuration?

File: tasks/configuration.yml (hunk 1 above)
Comment (a-alex-l) [inline tasks/configuration.yml:1]: There aren't any files

Comment (a-alex-l) [inline tasks/configuration.yml:1]: Do we really need to notify handlers if we have `--config.reload.automatic` option in service unit? I don't know, need testing

Comment (a-alex-l) [inline tasks/configuration.yml:1]: Where is this template?

File: templates/logstash.service.j2 [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# {{ ansible_managed }}
```
Comment (a-alex-l) [inline templates/logstash.service.j2:1]: Add `# {{ ansible_managed  }}` string to any template managed by Ansible. Read more at [config docs](https://jpmens.net/2020/09/29/using-ansible-managed/).

Note: don't add any dynamic value into that variable

File: tasks/install.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (a-alex-l) [inline tasks/install.yml:1]: Why we add user to that group?

File: templates/logstash.service.j2 (hunk 3 above)
Comment (a-alex-l) [inline templates/logstash.service.j2:1]: It's really bad practice to hardcode variables. Add them to defaults and put here

**Pull request closed without merging**